src/main/java/com/epam/pages/popup/CreatePopup.java

```diff
@@ -113,49 +120,63 @@ public boolean checkAllInputFieldsAreEmpty() {
     }
 
     public boolean checkGeneratedPasswordIsFilled() {
+        logger.info("Check password field is filled");
         return !passwordInput.getDomProperty("value").isEmpty();
     }
 
     public boolean checkGeneratePasswordButtonIsEnabled() {
+        logger.info("Check 'Generate password' button is active");
         return generatePasswordButton.isEnabled();
     }
 
     public boolean checkGeneratedPasswordStructure() {
+        logger.info("Check generated password structure");
         return passwordInput.getDomProperty("value")
-                .matches("(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$#!%*?&])[A-Za-z\\d@$!#%*?&]{9,50}");
+                .matches("(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$#!%*?&])[A-Za-z\\d@#$%&*()]{9,50}");
```

Comment:
In your regex you can also add other symbols, e.g. `~!^-_?+=|

Reply:
This regex has been written based on the requirement in the confluence page